web/blueprints/user/__init__.py

```diff
@@ -748,6 +768,57 @@ def default_response(refill_form_data: bool = False) -> ResponseReturnValue:
     return redirect(url_for(".user_show", user_id=user.id))
 
 
+@bp.route('/<int:user_id>/delete_membership/<int:membership_id>/', methods=['GET', 'POST'])
+@access.require('groups_change_membership')
+def membership_delete(user_id: int, membership_id: int) -> ResponseReturnValue:
+    membership = get_membership_or_404(membership_id)
+    assert isinstance(membership.group, PropertyGroup)
+    if membership.group.permission_level > current_user.permission_level:
+        flash("Eine Bearbeitung von Gruppenmitgliedschaften für Gruppen mit "
+              "höherem Berechtigungslevel ist nicht möglich.", 'error')
+        abort(403)
+    if session.utcnow() >= membership.active_during.begin:
+        flash("Nur Mitgliedschaften, die in der Zukunft liegen, können gelöscht werden."
+              , 'error')
+        abort(403)
+    form = FlaskForm()
+
+    def default_response() -> ResponseReturnValue:
+        form_args = {
+            'form': form,
+            'cancel_to': url_for('user.user_show', user_id=membership.user_id, _anchor='groups'),
+            'submit_text': 'Löschen',
+            'actions_offset': 0
+        }
+
+        return render_template('generic_form.html',
+                               page_title=("Mitgliedschaft {} für "
+                                           "{} löschen".format(
+                                                membership.group.name,
+                                                membership.user.name)),
+                               membership_id=membership_id,
+                               user=membership.user,
+                               form=form,
+                               form_args=form_args
+                               )
```

Comment:
Some formatting concerns I have here can be fixed easily by applying `darker`.
Did you do `pre-commit install`? (the setup script should have done it for you)

Reply:
Ran darker manually on it. Only did reformatting after I moved the first line.

Reply:
Nevermind, the pre-commit hook seems to be broken on my end and darker still complains in the GitHub checks.

Reply:
Fixed now